plugins/out_parseable/parseable.c

```diff
@@ -0,0 +1,378 @@
+                                        NULL);
+
+    ctx->upstream->base.net.connect_timeout = ctx->connect_timeout;
+    ctx->upstream->base.net.accept_timeout = ctx->accept_timeout;
+
+    flb_plg_info(ctx->ins, "Timeouts - Connect: %ds, Accept: %ds", ctx->connect_timeout, ctx->accept_timeout);
+
+    /* Set retry limit */
+    char retry_limit_str[16];
+    snprintf(retry_limit_str, sizeof(retry_limit_str), "%d", ctx->retry_limit);
+    flb_output_set_property(ins, "Retry_Limit", retry_limit_str);
+    
+    flb_plg_info(ctx->ins, "Retry limit set to: %d", ctx->retry_limit);
+
+    if (!ctx->upstream) {
+        flb_free(ctx);
+        return -1;
+    }
+
+    /* Export context */
+    flb_output_set_context(ins, ctx);
+
+    return 0;
+}
+
+/* Main flush callback */
+static void cb_parseable_flush(struct flb_event_chunk *event_chunk,
+                             struct flb_output_flush *out_flush,
+                             struct flb_input_instance *i_ins,
+                             void *out_context,
+                             struct flb_config *config)
+{
+    struct flb_out_parseable *ctx = out_context;
+    struct flb_log_event_decoder log_decoder;
+    struct flb_log_event log_event;
+    struct flb_record_accessor *ra = NULL;
+    struct flb_record_accessor *ns_ra = NULL;  // For checking namespace
+    (void) config;
+    struct flb_http_client *client;
+    struct flb_connection *u_conn;
+    flb_sds_t body;
+    flb_sds_t x_p_stream_value = NULL;
+    int ret;
+    int i;
+    size_t b_sent;
+    msgpack_sbuffer sbuf;
+    msgpack_packer pk;
+
+    /* Initialize event decoder */
+    flb_plg_info(ctx->ins, "Initializing event decoder...");
+    ret = flb_log_event_decoder_init(&log_decoder, (char *) event_chunk->data, event_chunk->size);
+    if (ret != FLB_EVENT_DECODER_SUCCESS) {
+        flb_plg_error(ctx->ins, "Failed to initialize event decoder");
+        FLB_OUTPUT_RETURN(FLB_ERROR);
+    }
+
+    /* Create record accessor if stream is set to $NAMESPACE */
+    if (ctx->stream && strcmp(ctx->stream, "$NAMESPACE") == 0) {
+        ra = flb_ra_create("$kubernetes['namespace_name']", FLB_TRUE);
+        if (!ra) {
+            flb_plg_error(ctx->ins, "Failed to create record accessor");
+            flb_log_event_decoder_destroy(&log_decoder);
+            FLB_OUTPUT_RETURN(FLB_ERROR);
+        }
+    }
+
+    /* Create record accessor for namespace exclusion check */
+    if (ctx->exclude_namespaces) {
+        ns_ra = flb_ra_create("$kubernetes['namespace_name']", FLB_TRUE);
+        if (!ns_ra) {
+            flb_plg_error(ctx->ins, "Failed to create namespace record accessor");
+            if (ra) {
+                flb_ra_destroy(ra);
+            }
+            flb_log_event_decoder_destroy(&log_decoder);
+            FLB_OUTPUT_RETURN(FLB_ERROR);
+        }
+    }
+
+    /* Process each event */
+    flb_plg_info(ctx->ins, "Processing events...");
```

Comment:
Please downgrade this and any other "overly verbose" messages to `debug` or `trace` level.